x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/LifecycleSettings.java

```diff
@@ -17,6 +19,8 @@
     public static final String LIFECYCLE_INDEXING_COMPLETE = "index.lifecycle.indexing_complete";
 
     public static final String SLM_HISTORY_INDEX_ENABLED = "slm.history_index_enabled";
+    public static final String SLM_RETENTION_SCHEDULE = "slm.retention_schedule";
```

Comment:
I'm definitely not in love with it either, I'm not sure yet about the `deletion_window_schedule` since it's not window-based (yet), let's revisit in the future?

Reply:
++, like I said this can be changed in a follow-up PR once we come up with a better name